recipes/xoshiro-cpp/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "1.1.0":
```

```suggestion
  "1.1":
```
The version of upstream is "1.1".
Please follow.

https://github.com/Reputeless/Xoshiro-cpp/releases/tag/v1.1